packages/user/Invite/test/src/Invite-test.cpp

```diff
@@ -62,16 +63,16 @@ namespace
        "qgsA/4UaAk39mbIe9/5Cx3m+A4ihRANCAAQEhhN0gTGxIbMxT6ZKGPEN4Xu+m+Ox\n"
        "cGky3DiVxoHkPFfGo81ZkIDBk3NOMSFiw5hZiscqwtY464TFBKdc8evq\n"
        "-----END PRIVATE KEY-----\n");
+
+   uint32_t inviteId = 1;
 }  // namespace
 
 // Helper functions for tests
 auto createInvite = [](auto& user, const auto& pubKey)
-{ return user.createInvite(pubKey, std::nullopt, std::nullopt, std::nullopt, std::nullopt); };
+{ return user.createInvite(++inviteId, pubKey, 1, false, ""); };
```

Comment:
This will cause slightly different behavior depending on which tests are run in what order, because `inviteId` is global.